packages/go_router/lib/src/route_data.dart

```diff
@@ -390,6 +390,28 @@ class TypedGoRoute<T extends GoRouteData> extends TypedRoute<T> {
   final List<TypedRoute<RouteData>> routes;
 }
 
+/// A superclass for each typed go route descendant
+@Target(<TargetKind>{TargetKind.library, TargetKind.classType})
```

Comment:
Can you also create a go_router_builder pr that shows how this will be used?

Reply:
gotcha

Reply:
it's #7174 @chunhtai , and I also updated the PR desc to reflect that

Reply:
@chunhtai I've removed this class as discussed in the #7174